examples/demo_qemu/wireguard_peer/peer1/wg0.conf

```diff
@@ -0,0 +1,9 @@
+[Interface]
+PrivateKey = 6FewnBCIntUTmmdvtaYvpQO5chP9gYhTmgf3whp0oE4=
```

Comment:
Could you add a warning comment here, same as [here](https://github.com/XmasHackathon2023/esp_wireguard/pull/3/files#r1760990648)